hadoop-plugin-test/src/main/gradle/positive/emergentFlow.gradle

```diff
@@ -0,0 +1,68 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath files("${project.pluginTestDir}/hadoop-plugin-${project.version}.jar", "${project.pluginTestDir}/hadoop-plugin-${project.version}-SNAPSHOT.jar")
+    // Must manually specify snakeyaml for testing, not required for users
+    classpath 'org.yaml:snakeyaml:1.18'
+  }
+}
+
+apply plugin: com.linkedin.gradle.hadoop.HadoopPlugin
+
+// Flow for testing yaml creation for emergent flows for Flow 2.0
+
+hadoop {
+  buildPath "jobs/emergentFlow"
+  cleanPath false
+
+  generateYamlOutput true
+
+  namespace('otherNamespace') {
+    propertyFile('emergentProperties') {
+      set properties: [
+              'new-parameter' : 'new-value',
+              'flow-level-parameter' : 'value2'
+      ]
+    }
+  }
+
+  workflow('emergentFlow') {
+    propertyFile('properties') {
+      set properties: [
+              'flow-level-parameter' : 'value'
+      ]
+    }
+
+    commandJob('shellBash') {
+      uses 'bash ./sample_script.sh'
+    }
+
+    commandJob('shellPwd') {
+      uses 'pwd'
+    }
+
+    commandJob('shellEcho') {
+      uses 'echo "This is an echoed text."'
+    }
+
+    targets 'shellPwd', 'shellEcho', 'shellBash'
+  }
+}
+
+hadoopZip {
+  libPath = "lib"
+  base {
+    from { fileTree("jobs/emergentFlow").exclude("otherNamespace").files } {
+      into "."
+    }
+  }
+  zip("emergentFlowTest") {
```

Comment:
How about adding two namespace in the test so that it would better demonstrate your fix?